docs/adding-a-page.md

```diff
@@ -98,7 +98,7 @@ export default (
 
 ### Starting the server
 
-Now that the component is setup we can run `npm start dev:search` and navigate to
+Now that the component is setup we can run `yarn dev:search` and navigate to
```

Comment:
https://github.com/mozilla/addons-frontend/issues/2223 would be a good place to fix up this guide